syz-verifier/monitoring_api.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2021 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+)
+
+// Monitor provides http based data for the syz-verifier monitoring.
+type Monitor struct {
+	externalStats *Stats
+}
+
+// MakeMonitor creates the Monitor instance.
+func MakeMonitor() *Monitor {
+	return &Monitor{}
+}
+
+// InitHTTPHandlers initializes the API routing.
+func (monitor *Monitor) InitHTTPHandlers() {
+	http.Handle("/api/stats.json", &statsHandler{
```

Comment:
It's more common to use a method as a handler and that's what we use in all other tools:
```
http.Handle("/api/stats.json", monitor.handleStats)
```
It's just less boilerplate and paperwork.
Any particular reason you prefer to use a separate handler object here?

Reply:
No special reason. Just followed the documentation and got this result. Will try method.